samples/Repos/Watching/Unwatch.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Unwatch where
```

Comment:
Could you please add this to the executables in `samples/github-samples.cabal` so that it gets built by the CI?
(You will likely have to delete the module header or change it to `Main` to get ghc to build the executable.)